tests/containers/buildx.pm

```diff
@@ -0,0 +1,66 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: buildx
+# Summary: test buildx plugin
+# - install buildx
+# - build test image
+# - run container with test image
+# - cleanup
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils;
+use version_utils qw(is_transactional);
+use transactional qw(trup_call check_reboot_changes);
+
+my $test_image = "test_buildx";
+my $test_container = "test_buildx";
+
+sub run {
+    select_serial_terminal;
+
+    my $docker_info = script_output("docker info");
+    record_info('Docker info pre-install', $docker_info);
```

Comment:
Just use `containers:install_docker_when_needed()` which handles all the corner cases and also outputs the `docker info`.